pkg/source/vmware/client.go

```diff
@@ -247,6 +246,9 @@ func (c *Client) IsPoweredOff(vm *migration.VirtualMachineImport) (bool, error)
 }
 
 func (c *Client) GenerateVirtualMachine(vm *migration.VirtualMachineImport) (*kubevirt.VirtualMachine, error) {
+	var boolFalse = false
+	var boolTrue = true
```

Comment:
Same question here.

Reply:
same as before, we can leave it here for now since there is no need to increase the scope for this variable